packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,344 @@
+
+```ts
+export const bundleToHashMappings = {
+  "all": "all-5623e5a4aaceb8ac",
+  "base/base": "base/base",
+  "base/base-globant": "base/base-globant-2e35f235ecacc6c3",
+  "base/icons": "base/icons-137edfdbd4d60c97",
+  // and so on ...
+```
+
+- `registerMercury`: (TODO)
+  <br>
+
+- `getImagePathCallbackDefinitions`: An object used to register the `getImagePathCallback` definitions for all controls in Chameleon, which is a property that is present on several Chameleon controls, and it specifies a callback that is executed when the path for an item image needs to be resolved.
+  <br>
+
+- `registryProperty`: (TODO)
+  <br>
+
+- `defineCustomElements`: registers all Chameleon Web Components in the browser’s customElements registry so they can be used in HTML.
+
+</details>
+
+> [!IMPORTANT]
+> Include and run `setBundleMapping` and `bundleToHashMappings` before using any Mercury or [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) utilities!
+
+#### Alternative: Using already generated CSS bundles
+
+> [!WARNING]
+> We don't recommend this way, because the generated CSS contains fixed paths for the assets and the CSS names don't contain a hash, which leads to cache issues when re-deploying your application after updating your version of Mercury.
+> We only include this case, because in some scenarios it can facilitate the initialization with Mercury.
+
+Copy the `<path to node_modules>/@genexus/mercury/dist/bundles/css` content to `{{ CSS outDir path }}` (or the CSS outDir path that you chose).
+
+### 3.2 Include Icons
+
+`app.ts` <small>(Suggested file)</small>
+
+```ts
+import type { ThemeModel } from "@genexus/chameleon-controls-library";
+import { bundleToHashMappings } from "{{ CSS outDir path }}/bundle-to-hash-mappings";
+
+const iconsBundle: ThemeModel = [
+  {
+    name: "mercury/base/icons",
+    url: `{{ Icons final path }}`${bundleToHashMappings["base/icons"]}.css`
+  }
+];
+
+// Use the iconsBundle in the ch-theme/ChTheme component...
+```
+
+`app.html` <small>(Suggested file)</small>
+
+```
+<ch-theme [model]="iconsBundle"></ch-theme>
+```
+
+<details>
+<summary>Code explained:</summary>
+<br>
+
+In the [3.1. Import declarations](#31-import-declarations) section we registered utilities to compute the icons in the component. The only remaining thing to do is download the path for the icons. To do this, we need to download a CSS bundle called `base/icons` using the `ch-theme` component.
+
+The only difference, is that the bundle is not downloaded by loading a **Theme**, instead of using the `getBundles` util.
+
+</details>
+
+### 3.3. Style the base application
+
+> [!TIP]
+> For performance reason, we recommend to style the base application, using a `link` tag in the `head` of the HTML, or even inlining the base CSS into the HTML.
+
+`index.html`
+
+```html
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="dark">
+  <!-- use "dark" or "light" theme -->
```

Comment:
The dark and light mode must be properly explained in a section, otherwise this comment will be missed the 90% of the time.